source/Government.h

```diff
@@ -54,6 +54,21 @@ class Government {
 		const Conversation *customDeathSentence = nullptr;
 	};
 
+	enum class SpecialPenalty : int {
+		NONE = 0,
+		PROVOKE,
+		ATROCITY
+	};
+
+	class PenaltyEffect {
+	public:
+		double reputationChange;
+		SpecialPenalty specialPenalty;
+		PenaltyEffect(double reputationChange = 0., SpecialPenalty specialPenalty = SpecialPenalty::NONE) :
+			reputationChange(reputationChange), specialPenalty(specialPenalty)
+		{}
```

Comment:
One more thing: constructor goes first.
```suggestion
		PenaltyEffect(double reputationChange = 0., SpecialPenalty specialPenalty = SpecialPenalty::NONE)
			: reputationChange(reputationChange), specialPenalty(specialPenalty)
		{}

		double reputationChange;
		SpecialPenalty specialPenalty;
```